### PR TITLE
docs(kfp): add local option to KFP installation overview

### PR DESCRIPTION
**Pull request by Bobgy**

### CODE REVIEW

File: content/en/docs/pipelines/installation/overview.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ all Kubeflow components and more integration with each platform.
 1. Do you want to use other Kubeflow components in addition to Pipelines?
 
     If yes, choose the [full Kubeflow deployment](#full-kubeflow-deployment).
+1. Can you use a cloud/on-prem Kubernetes cluster?
+
+    If no, you can try Kubeflow Pipelines on a local Kubernetes cluster following [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment) for learning and testing purposes.
```
Comment (8bitmp3) [inline content/en/docs/pipelines/installation/overview.md:26]:
```suggestion
    If you can't, you should try using Kubeflow Pipelines on a local Kubernetes cluster for learning and testing purposes by following the steps in [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment).
```

WDYT? cc @alfsuse @RFMVasconcelos 

Thanks @Bobgy

Comment (Bobgy) [reply content/en/docs/pipelines/installation/overview.md:26]: LGTM! Let's wait for their opinions too

Comment (alfsuse) [reply content/en/docs/pipelines/installation/overview.md:26]: /lgtm sorry I was not much very active these days.

**Comment by Bobgy** [comment]
@8bitmp3 Accepted the suggestion, any other comments?

**Pull request merged**